project/Dependencies.scala

```diff
@@ -25,30 +25,30 @@ object Dependencies {
 
   lazy val CVMappingAll  = crossMapped("2.11.7" -> "2.11")
 
-  lazy val slf4jVersion = "1.7.2"
+  lazy val slf4jVersion = "1.7.25"
 
   // Compile scope:
   // Scope available in all classpath, transitive by default.
-  lazy val commons_codec          = "commons-codec"              % "commons-codec"      % "1.10"
-  lazy val commons_fileupload     = "commons-fileupload"         % "commons-fileupload" % "1.3.1"
+  lazy val commons_codec          = "commons-codec"              % "commons-codec"      % "1.11"
+  lazy val commons_fileupload     = "commons-fileupload"         % "commons-fileupload" % "1.3.3"
   lazy val commons_httpclient     = "commons-httpclient"         % "commons-httpclient" % "3.1"
   lazy val javamail               = "javax.mail"                 % "mail"               % "1.4.7"
   lazy val jbcrypt                = "org.mindrot"                % "jbcrypt"            % "0.4"
-  lazy val joda_time              = "joda-time"                  % "joda-time"          % "2.9.2"
-  lazy val joda_convert           = "org.joda"                   % "joda-convert"       % "1.8.1"
-  lazy val htmlparser             = "nu.validator.htmlparser"    % "htmlparser"         % "1.4"
-  lazy val mongo_java_driver      = "org.mongodb"                % "mongodb-driver"     % "3.6.3"
-  lazy val mongo_java_driver_async  = "org.mongodb"              % "mongodb-driver-async" % "3.6.3"
+  lazy val joda_time              = "joda-time"                  % "joda-time"          % "2.10"
+  lazy val joda_convert           = "org.joda"                   % "joda-convert"       % "2.0.2"
```

Comment:
Looks like a major version bump. Any concerns around that?

Reply:
Hmm, well the only thing this touches is the json-ext package, so none major from the framework's point of view. I guess in theory any of these could break transitive binary/API compatibility, but given how poorly developers tend to enforce semver it may be impossible to preserve that with _any_ dependency bump. :/

Reply:
To be clear, certainly worth investigating their change log when I get a chance though.